### PR TITLE
More Stretch -> Buster changes

### PR DESCRIPTION
**Pull request by lurch**
See also #1221 and #1215

### CODE REVIEW

File: glossary/README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@
 - **Polyfuse** - A self-resetting electrical fuse used on some Pi models to protect against over-current; can take hours or even days to reset.
 - **`raspi-config`** – A command line tool for configuring Raspbian's operating system features; see our [official documentation](../configuration/raspi-config.md).
 - **raspivid, raspistill, raspiyuv, raspividyuv** – a set of applications to run the Raspberry Pi Camera Module in various modes; see our [official documentation](../raspbian/applications/camera.md).
-- **Raspbian** - A Debian-based Linux distribution targeted at the entire Raspberry Pi range, regarded as the official operating; the current version is based on Debian Stretch, with the first release in August 2017; see the [Wikipedia entry for further release details](https://en.wikipedia.org/wiki/Raspbian).
+- **Raspbian** - A Debian-based Linux distribution targeted at the entire Raspberry Pi range, regarded as the official operating; the current version is based on Debian Buster, with the first release in June 2019; see the [Wikipedia entry for further release details](https://en.wikipedia.org/wiki/Raspbian).
```
Comment (andrum99) [inline glossary/README.md]: Official operating -> official operating system

Comment (lurch) [reply glossary/README.md]: Thanks :+1:

Comment (andrum99) [reply glossary/README.md]: No probs

**Comment by JamesH65** [comment]
Can you rebase this, it won't merge as currently stands.

**Comment by lurch** [comment]
Looks like GitHub had a hiccup here; but the changes from this PR _have_ been merged, even though the status is shown as "Closed" instead of "Merged".

**Pull request closed without merging**